x/data/client/testsuite/grpc.go

```diff
@@ -474,22 +472,22 @@ func (s *IntegrationTestSuite) TestQueryResolverInfo() {
 		expItems int
 	}{
 		{
-			"invalid url",
-			fmt.Sprintf("%s/regen/data/v1/resolver?url=%s", val.APIAddress, "foo"),
+			"invalid id",
+			fmt.Sprintf("%s/regen/data/v1/resolver?id=%d", val.APIAddress, 404),
 			true,
 			"not found",
 			0,
 		},
 		{
 			"valid request",
-			fmt.Sprintf("%s/regen/data/v1/resolver?url=%s", val.APIAddress, url),
+			fmt.Sprintf("%s/regen/data/v1/resolver?id=%d", val.APIAddress, s.resolverID),
 			false,
 			"",
 			2,
```

Comment:
Doesn't look like `expItems` is ever checked. Should we remove it from the table?

Also - I don't think we're actually able to query resolver with pagination, so maybe we should drop this last test case entirely?

Reply:
Good catch. Looks like some old copy pasta that went unnoticed.

Reply:
https://github.com/regen-network/regen-ledger/pull/1128/commits/56f1b975eeab03dc2338fe4a9b3de0e53fa24c0f